src/tests/functional/plugin/shared/src/behavior/ov_executable_network/hetero_properties.cpp

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_executable_network/hetero_properties.hpp"
+#include "openvino/runtime/properties.hpp"
+#include <cstdint>
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+void OVClassHeteroExecutableNetworkGetMetricTest::SetCpuAffinity(ov::Core& core, std::vector<std::string>& expectedTargets) {
+#ifdef ENABLE_INTEL_CPU
+    auto layermap = core.query_model(actualNetwork, heteroDeviceName);
+    for (auto &iter : layermap) {
+        if (iter.first.find("Concat") != std::string::npos)
+            layermap[iter.first] = CommonTestUtils::DEVICE_CPU;
+    }
+    for (auto& node : actualNetwork->get_ops()) {
+        auto affinity = layermap[node->get_friendly_name()];
+        node->get_rt_info()["affinity"] = affinity;
+    }
+    if (target_device.find(CommonTestUtils::DEVICE_CPU) == std::string::npos)
+        expectedTargets = {target_device, CommonTestUtils::DEVICE_CPU};
+#endif
+}
+
+TEST_P(OVClassHeteroExecutableNetworkGetMetricTest_SUPPORTED_CONFIG_KEYS, GetMetricNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
```

Comment:
Have you checked the function?

Reply:
Not yet, but I remember about it, I will do